client/src/pages/Pages.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Route } from 'react-router-dom' ;
+import LoginPage from './LoginPage';
+import SignUp from './SignUp';
+import Home from './Home';
+import NavBar from '../components/nav_comp/NavBar'
+import '../style/home.css'
+
+export default function Pages(){
+    return (
+        <>  
+            <NavBar />
+            <div className="root-container">
+                <Route path='/login' component={LoginPage} />
+                <Route path='/signup' component={SignUp} />
+                <Route path='/' component={Home} />
```

Comment:
I heard that some people stick with using single quotes in JavaScript code and double quotes for JSX code, so I started following that and suggest you do as well!

```suggestion
                <Route path="/login" component={LoginPage} />
                <Route path="/signup" component={SignUp} />
                <Route path="/" component={Home} />
```